feat: added v1.1.2 core upgrade docs (#1275)
axelarnetwork/axelar-docs: src/content/docs/resources/mainnet/v112.mdx

Reviewer: wrong folder, should be under `upgrades` folder, and rename file to `v1_1.mdx` since it's the `v1.1.x` release

> # Mainnet upgrade: V1.1.2

Reviewer: ```suggestion
# Mainnet upgrade: v1.1
```
title doesn't need to include the patch

Reviewer: make testnet doc consistent with changes as well

import { Callout } from "/src/components/callout";

Instructions for the `2024-Dec-18` mainnet upgrade to axelar-core `v1.1.2`.
Release notes can be found [here](https://github.com/axelarnetwork/axelar-core/releases/tag/v1.1.2)

> Upgrade height `15892300` [countdown](https://www.mintscan.io/axelar/block/15892300)

Reviewer: update the height

> ## About the V1.1.2 upgrade

Reviewer: ```suggestion
## About the v1.1.2 upgrade
```

> The `v1.1.2` upgrade is a non-consensus breaking patch release for `v1.1.x.` `v1.1.x.` Once live, it will enable the deployment of ITS Hub on mainnet.

Reviewer: ```suggestion
The `v1.1.2` upgrade is a consensus breaking release for Axelar mainnet. It enables the deployment of ITS Hub on mainnet.
```
the upgrade is happening from v1.0 -> v1.1 which is consensus breaking

1. If you're a validator or have delegated to one, please vote for the upgrade proposal via

```bash
axelard tx gov vote 257 yes --from validator
```

<Callout type="warning" emoji="⚠️">
  `axelard` must be built with `go-1.23` for this release.
</Callout>

> 2. Wait for the proposed upgrade block, `15892300`. Your node will panic at that block height with a log: `{"level":"error","module":"consensus","err":"UPGRADE \"v1.1.2\" NEEDED at height: ",`. Stop your node after chain halt.

Reviewer: update height
